src/utils/deleteContracts.ts

```diff
@@ -24,22 +26,25 @@ export default async function deleteContracts(
     "tfkvstore" as any
   );
 
-  // remove named vms
+  // remove deployment in namespace `type`
   console.log({ grid });
   const obj = await grid.connect().then(() => {
     return grid[key].delete({ name });
```

Comment:
I think we need to delete the gateway here too

Reply:
the grid client loaded here have `projectName = type` and any gateway deployment is deployed without `projectName`

Reply:
machine and gateway contracts should be on the same namespace.